app/src/main/java/net/squanchy/service/firebase/FirebaseDbService.java

```diff
@@ -143,4 +153,17 @@ private Completable updateFavorite(String eventId, Func1<DatabaseReference, Task
                     .addOnFailureListener(emitter::onError);
         });
     }
+
+    public Completable addAchievement(String achievementId, String userId, Long timestamp) {
```

Comment:
Reorder params to match usage order (`userId`, `achievementId`, `timestamp`) please